File: windowslib/src/main/resources/scala-native/windows/accCtrl/trusteeType.c
#if defined(_WIN32) || defined(WIN32)
#define WIN32_LEAN_AND_MEAN
#include <accctrl.h>

int scalanative_trustee_is_unknown() { return TRUSTEE_IS_UNKNOWN; }
int scalanative_trustee_is_user() { return TRUSTEE_IS_USER; }
int scalanative_trustee_is_group() { return TRUSTEE_IS_GROUP; }
int scalanative_trustee_is_domain() { return TRUSTEE_IS_DOMAIN; }
int scalanative_trustee_is_alias() { return TRUSTEE_IS_ALIAS; }
int scalanative_trustee_is_well_known_group() {
    return TRUSTEE_IS_WELL_KNOWN_GROUP;
}
int scalanative_trustee_is_deleted() { return TRUSTEE_IS_DELETED; }
int scalanative_trustee_is_invalid() { return TRUSTEE_IS_INVALID; }
int scalanative_trustee_is_computer() { return TRUSTEE_IS_COMPUTER; }

#endif // defined(_WIN32)

File: windowslib/src/main/scala/scala/scalanative/windows/accctrl/TrusteeType.scala
package scala.scalanative.windows.accctrl

import scala.scalanative.unsafe.*

@extern
object TrusteeType {
  @name("scalanative_trustee_is_unknown")
  def TRUSTEE_IS_UNKNOWN: TrusteeType = extern
  @name("scalanative_trustee_is_user")
  def TRUSTEE_IS_USER: TrusteeType = extern
  @name("scalanative_trustee_is_group")
  def TRUSTEE_IS_GROUP: TrusteeType = extern
  @name("scalanative_trustee_is_domain")
  def TRUSTEE_IS_DOMAIN: TrusteeType = extern
  @name("scalanative_trustee_is_alias")
  def TRUSTEE_IS_ALIAS: TrusteeType = extern
  @name("scalanative_trustee_is_well_known_group")
  def TRUSTEE_IS_WELL_KNOWN_GROUP: TrusteeType = extern
  @name("scalanative_trustee_is_deleted")
  def TRUSTEE_IS_DELETED: TrusteeType = extern
  @name("scalanative_trustee_is_invalid")
  def TRUSTEE_IS_INVALID: TrusteeType = extern
  @name("scalanative_trustee_is_computer")
  def TRUSTEE_IS_COMPUTER: TrusteeType = extern
}
